src/value.rs

```diff
@@ -356,7 +405,9 @@ impl ValueCommitment {
             pallas::Scalar::from(abs_value)
         };
 
-        ValueCommitment(V * value + R * rcv.0)
+        let V_zsa = asset.cv_base();
+
+        ValueCommitment(V_zsa * value + R * rcv.0)
```

Comment:
str4d ([Link](https://github.com/zcash/orchard/pull/471#discussion_r2542846302)): This is not necessarily `V_zsa` in this context; we should continue to use the existing naming to match the protocol spec.
```suggestion
        ValueCommitment(V * value + R * rcv.0)
```